gql/transport/local_schema.py

```diff
@@ -50,27 +50,16 @@ async def execute(
     async def subscribe(
         self, document: DocumentNode, *args, **kwargs,
     ) -> AsyncGenerator[ExecutionResult, None]:
-        """Send a query and receive the results using an async generator
-
-        The query can be a graphql query, mutation or subscription
+        """Send a subscription and receive the results using an async generator
 
         The results are sent as an ExecutionResult object
         """
 
-        subscribe_result = subscribe(self.schema, document, *args, **kwargs)
+        subscribe_result = await subscribe(self.schema, document, *args, **kwargs)
 
         if isinstance(subscribe_result, ExecutionResult):
-            yield ExecutionResult
+            yield subscribe_result
```

Comment:
How was that working before? Anyway, good catch!

Reply:
Yes, that was a typo but it happened only in case of errors with subscriptions using the local_schema transport. So it was not really used apparently.
That shows the importance of test coverage!